lib/anycable/nsq.rb

```diff
@@ -0,0 +1,18 @@
+module AnyCable
+  module NSQ
+  end
+end
+
+require "anycable/nsq/server"
+
+# GRPC example
+
+# AnyCable.server_builder = ->(config) {
```

Comment:
Any reason to keep this?  Not like it hurts anything, just curious.

Reply:
None that I can think of... I can delete it to avoid confusion (it can still be referenced in the `lib/anycable/grpc.rb` file.